mautic/mautic
# fix: dreprecated from (self::MASTER_REQUEST) to (self::MAIN_REQUEST)

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Navigate to the app/middlewares/Dev/IpRestrictMiddleware.php directory.
3. press ctrl-f in your code editor environment (Vs code) to find self::MASTER_REQUEST which is the deprecated constant.


### Steps taken to resolve this problem on all located files:

1. search for the constant self::MASTER_REQUEST
2. replaced with self::MAIN_REQUEST
